.github/workflows/build.yml

```diff
@@ -1,13 +1,81 @@
-name: Build
+name: build # should probably be renamed
```

Comment:
Any ideas for a name here? @Benzammour @mormod

Reply:
quality-control?